lib/services.js

```diff
@@ -2,7 +2,12 @@ var fs = require('fs');
 var ejs = require('ejs');
 
 ejs.filters.q = function(obj) {
-  return JSON.stringify(obj, null, 2 );
+  // to handle strings being returned with double-quotes
+  // but linter requires single quotes for strings
+  if (typeof obj == 'string' && !/\\'/.test(obj))
+    return '\'' + obj + '\'';
+  else
+    return JSON.stringify(obj, null, 2);
```

Comment:
It's fine, just update the comment with a clearer description. You have the why, but also include the what (as in what is the context - this code case is here for the generated file, blah blah).